### PR TITLE
Feature/add home page

### PR DESCRIPTION
**Pull request by anthonymckerron**

### CODE REVIEW

**Comment by anthonymckerron** [comment]
Will add to the contributors section in a new branch.

File: src/layout/about/features.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+		PropTypes.shape({
+			title: PropTypes.string,
+			text: PropTypes.string,
+		})
+	).isRequired,
+};
+
+Features.defaultProps = {};
+
+export default Features;
```
Comment (sukhrajghuman) [inline src/layout/about/features.js]: this file is the same as `layout/home/features.js`.
No need to replicate it

File: src/layout/about/promo.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+			imgalt: PropTypes.string,
+			url: PropTypes.string,
+			text: PropTypes.string,
+		})
+	).isRequired,
+};
+
+Promo.defaultProps = {};
+
+export default Promo;
```
Comment (sukhrajghuman) [inline src/layout/about/promo.js]: This file is a duplicate. Delete it at one of the locations.

File: src/layout/about/users.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+);
+
+
+Users.propTypes = {
+
+};
+
+Users.defaultProps = {};
+
+export default Users;
```
Comment (sukhrajghuman) [inline src/layout/about/users.js]: This file is duplicated. Delete it at one of the locations

File: src/layout/about/promo.js (hunk 2 above)
Comment (anthonymckerron) [reply src/layout/about/promo.js]: Done

**Pull request merged**